doc/api/util.md

```diff
@@ -1973,13 +1978,19 @@ changes:
     the built-in behavior, from adding additional checks through to reprocessing
     the tokens in different ways.
     **Default:** `false`.
+  * `help` {string} General help text to display at the beginning of help output.
+  * `enableHelpPrinting` {boolean} When `true`, if any options have help text
```

Comment:
From the description and implementation, `enableHelpPrinting` appears to display the help and exit whatever the arguments. That doesn't match what was suggested in original issue.

The intent of the `enableHelp` mentioned in the original issue was to effectively add an `-h, --help` entry and display the help _only_ if the option was supplied. In other words, fully implement the simple case. But make it opt-in so it is not a breaking change for existing users of `parseArgs`.

If an author wants different options flags, including translations, they would add the option themselves and display the help text themselves if the option was supplied. (You might have been aiming for that use case.)

Reply:
I really misunderstood that, I thought that only the flag would be responsible for showing help messages (when they are present).
I'll make adjustments to check the `-h, -help` flags, thanks